web/src/games/secretcodes/PlayBoard.tsx

```diff
@@ -57,7 +58,8 @@ export class PlayBoard extends React.Component<IPlayBoardProps, IPlayBoardState>
     return this.props.G.teams[this._playerID()];
   }
 
-  _showSpymasterView = (): boolean => isPlayerSpymaster(this.props.G, this._playerID()) && this.state.spymasterView;
+  _showSpymasterView = (): boolean =>
+    this.props.isGameOver || (isPlayerSpymaster(this.props.G, this._playerID()) && this.state.spymasterView);
```

Comment:
Only this is not enough, you need to change this file:
https://github.com/freeboardgames/FreeBoardGames.org/blob/a0c321f4c32f8c0e80ae93450635d0483f6f3f5a/web/src/games/secretcodes/game.ts#L41

to also return G if the game is over (this.props.ctx.winner is truthy).